sqlserver/tests/test_integration.py

```diff
@@ -568,6 +571,15 @@ def test_resolved_hostname_set(
         for k, v in cloud_metadata.items():
             instance_docker[k] = v
     instance_docker['dbm'] = dbm_enabled
+    instance_docker['procedure_metrics'] = {'enabled': False}
```

Comment:
if query metrics is enabled, is there a reason to disable proc metrics? Asking since its enabled by default, so maybe it _should_ be tested in integration tests 

(although i have no idea if this makes sense, see my question below)